Disher/Models/List.h

```diff
@@ -12,9 +12,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface List : PFObject<PFSubclassing>
 
-@property (nonatomic, strong) NSNumber *listID;
+@property (nonatomic, strong) NSString *listName;
```

Comment:
curious why replacing ID with name, instead of having both? how would we handle duplicate named lists?

Reply:
or does `PFSubclassing` have `objectID`?

Reply:
yes; I started using objectId which is automatically generated for each object in Parse